stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/list/Channels.kt

```diff
@@ -60,3 +69,19 @@ public fun Channels(
         }
     }
 }
+
+/**
+ * Represents an almost invisible dummy item to be added to the top of the list.
+ *
+ * If the list is scrolled to the top and a channel new item is added or moved
+ * to the position above, then the list will automatically autoscroll to it.
+ */
+@Composable
+private fun DummyFirstChannelItem() {
+    Box(
+        modifier = Modifier
+            .height(1.dp)
+            .fillMaxWidth()
+            .background(color = ChatTheme.colors.appBackground)
```

Comment:
I wouldn't put a background color on this as we can't guarantee that users won't change the background color of the `Channels` component or that their app background won't be a non `ChatTheme` color for some reason.

In general I think it can only cause headaches with no benefit.

I don't know if this was introduced to solve a specific issue though.

Reply:
Hoisted the background color to the channels screen